new-pipeline/src/evo.js

```diff
@@ -505,10 +504,9 @@ function calculateEvolutions(callback) {
   channelVersions.forEach(function (channelVersion) {
     var parts = channelVersion.split("/"); //wip: fix this
     getHistogramEvolutionLines(parts[0], parts[1], measure, aggregates,
-      filterSets, $("input[name=sanitize-toggle]:checked")
-      .val() !== "0", $("input[name=build-time-toggle]:checked")
-      .val() !== "0",
-      function (newLinesMap, newDescription) {
+      filterSets, $("input[name=sanitize-toggle]").is(":checked")
+      , $("input[name=build-time-toggle]").is(":checked")
```

Comment:
The style in this file is to put the commas at the ends of the previous line, not at the beginning of the next line.